opennms-services/src/main/java/org/opennms/netmgt/vacuumd/AutomationProcessor.java

```diff
@@ -33,14 +33,7 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Date;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
avoid wildcard imports.

Reply:
This part seems Intellij automatically changed it. I can change it back.

Reply:
Okay, I think I'll just respect Intellij and leave this alone if you are okay with it. Because when I do code -> optimize imports in Intellij, it's getting even "worse"(more "*" :P)

Reply:
Try setting up Intellij with these settings https://github.com/OpenNMS/ide-setup 
 Think that should resolve it.  More here https://wiki.opennms.org/wiki/IDEA_and_OpenNMS

Reply:
Yes. This works. Updated. Thanks.